README.md

```diff
@@ -78,114 +78,97 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
-An experimental version of [PowerToys (v0.46)][github-prerelease-link] was released the week of September 13th. Download and install it to try out the latest improvements to our Video Conference Mute utility! All updates from the v0.45.0 release will still apply in v0.46.0.
+An experimental version of PowerToys (v0.48) will be released the week of October 4th, introducing improvements to our Video Conference Mute utility! All updates from the v0.47.0 release will still apply in v0.48.
 
-#### Highlights from v0.45
+#### Highlights from v0.47
 
 **General**
 
-- Settings and OOBE windows updated with Fluent UX! We hope you enjoy the new modern feel of the application menus as we align our product with upcoming Windows 11 interfaces. Special thanks to @niels9001 for driving the development of this UI, along with many thanks to the various community members who offered constant feedback and adjustments to make this a truly spectacular update!
-- Added button to settings which displays version history. Thanks @niels9001!
-- Signed PowerToysSetupCustomActions.dll.
-- Improved auto-update experience. Thanks @niels9001!
-- Aligned OOBE theme color with Settings theme color. Thanks @niels9001!
-- Adjusted labeling of "Restart as Administrator" button to "Restart PowerToys as Administrator" to avoid ambiguity in meaning. Thanks @niels9001!
-- Added colored icons to settings sidebar. Thanks @niels9001!
-- Fixed accessibility issue in OOBE where Microsoft Docs and PowerToys release notes links could not be navigated to via keyboard. Thanks @niels9001!
-- Fixed settings header alignment. Thanks @niels9001!
-- Fixed text under updates section to be visible when in light mode. Thanks @niels9001!
-- Updated "Learn More" text to be more descriptive. Thanks @niels9001!
-- Updated "Read more" text on updates to be more descriptive. Thanks @niels9001!
-- Added link to documentation in system tray. Thanks @BenConstable9!
-- Fixed error caused by file in use issues when installing PowerToys.
-- Fixed issue where opening settings from start menu didn't work when PowerToys was run as admin. Thanks @davidegiacometti!
+- Fixed issue with new updates changing the PowerToys install location.
+- Fixed settings with NumberBox elements overlapping the delete button.
+- Fixed issue with the bug report tool not generating .zip files.
+- Fixed issue with disabled shortcuts looking the same as when enabled. Thanks @niels9001!
+- Fixed inconsistent with of sidebar icons. Thanks @niels9001!
+- Fixed sidebar UI not scaling for longer text strings in certain localizations. Thanks @niels9001!
+- Fixed issue with settings not displaying invalid keystroke assignments. Thanks @niels9001!
+- Added user defined shortcuts when set to the "Welcome to PowerToys" instead of the default shortcuts.
```

Comment:
On the last readme update we discussed that we want to split setting and general into two sections.

Reply:
We considered splitting the two sections last release due to the large number of changes introduced with the new UI. Given the smaller set of updates, it's fine to leave them under the general section this time around.